Library/Formula/gabedit.rb

```diff
@@ -16,4 +15,8 @@ def install
     system "make", *args
     bin.install "gabedit"
   end
+
+  test do
+    assert (bin/"gabedit").exist?
+  end
```

Comment:
We try to avoid tests that just assert that the files were installed, so let's omit it if there's nothing better.